db/migrate/20190805204523_drop_deprecated_columns.rb

```diff
@@ -0,0 +1,47 @@
+class DropDeprecatedColumns < ActiveRecord::Migration[5.1]
+  def change
+    remove_column :pipeline_run_stages, :failed_jobs
+
+    remove_column :host_genomes, :skip_deutero_filter
+
+    remove_column :metadata_fields, :validation_type
+
+    remove_column :pipeline_runs, :version
```

Comment:
@boris-dimitrov, the `version` last blames to #1189, and this col has not been set since 2018-07-09... should we drop it?

Reply:
I don't recall precisely how the design has evolved since the time I wrote this in April 2018.     Nevertheless, the code line
```
pr.version = `aws s3 cp #{version_s3_path} -`
```
is what used to set this.   If that line is no longer executed, then you can probably remove that columns.   I am sure you will modify any references to it in the codebase to assume NULL.

Reply:
I replaced all callsites of `version` with `pipeline_version`.